examples/bootstrap-next/README.md

```diff
@@ -0,0 +1,15 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) and includes [Mirrorful](https://www.mirrorful.com/).
+
+## Getting Started
+
+First, run the development server and install packages
+
+```
+yarn
+```
+
+```bash
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
Could you copy the other example READMEs to explain how to run mirrorful? @semijacks

Reply:
Alright. I'll do that